From 58c8d1b1bec9d1ac05db2007f9f59527cbe03b55 Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <yasufum.o@gmail.com>
Date: Wed, 31 Jul 2019 16:15:44 +0900
Subject: [PATCH 1/6] spp_vf: rename file classifier_mac

This update is to change files named as `classifier_mac` to
`classifier` because claffisication only supports MAC and `_mac` has
no meaning.

Signed-off-by: Yasufumi Ogawa <yasufum.o@gmail.com>
---
 src/vf/Makefile                           | 2 +-
 src/vf/{classifier_mac.c => classifier.c} | 2 +-
 src/vf/{classifier_mac.h => classifier.h} | 0
 src/vf/spp_vf.c                           | 2 +-
 src/vf/vf_cmd_runner.c                    | 2 +-
 5 files changed, 4 insertions(+), 4 deletions(-)
 rename src/vf/{classifier_mac.c => classifier.c} (99%)
 rename src/vf/{classifier_mac.h => classifier.h} (100%)

diff --git a/src/vf/Makefile b/src/vf/Makefile
index 92d2854..5981d62 100644
--- a/src/vf/Makefile
+++ b/src/vf/Makefile
@@ -15,7 +15,7 @@ SPP_SEC_DIR = ../shared/secondary
 SPP_WKT_DIR = ../shared/secondary/spp_worker_th
 
 # all source are stored in SRCS-y
-SRCS-y := spp_vf.c classifier_mac.c forwarder.c
+SRCS-y := spp_vf.c classifier.c forwarder.c
 SRCS-y += $(SPP_SEC_DIR)/string_buffer.c
 SRCS-y += $(SPP_SEC_DIR)/json_helper.c
 SRCS-y += $(SPP_SEC_DIR)/utils.c $(SPP_SEC_DIR)/add_port.c
diff --git a/src/vf/classifier_mac.c b/src/vf/classifier.c
similarity index 99%
rename from src/vf/classifier_mac.c
rename to src/vf/classifier.c
index f3be5b6..49c7d01 100644
--- a/src/vf/classifier_mac.c
+++ b/src/vf/classifier.c
@@ -21,7 +21,7 @@
 #include <rte_launch.h>
 #include <netinet/in.h>
 
-#include "classifier_mac.h"
+#include "classifier.h"
 #include "spp_vf.h"
 #include "shared/secondary/return_codes.h"
 #include "shared/secondary/string_buffer.h"
diff --git a/src/vf/classifier_mac.h b/src/vf/classifier.h
similarity index 100%
rename from src/vf/classifier_mac.h
rename to src/vf/classifier.h
diff --git a/src/vf/spp_vf.c b/src/vf/spp_vf.c
index caf4678..2091023 100644
--- a/src/vf/spp_vf.c
+++ b/src/vf/spp_vf.c
@@ -7,7 +7,7 @@
 #include <getopt.h>
 
 #include "spp_vf.h"
-#include "classifier_mac.h"
+#include "classifier.h"
 #include "forwarder.h"
 #include "shared/secondary/utils.h"
 #include "shared/secondary/spp_worker_th/cmd_utils.h"
diff --git a/src/vf/vf_cmd_runner.c b/src/vf/vf_cmd_runner.c
index 16328f4..9536a80 100644
--- a/src/vf/vf_cmd_runner.c
+++ b/src/vf/vf_cmd_runner.c
@@ -2,7 +2,7 @@
  * Copyright(c) 2019 Nippon Telegraph and Telephone Corporation
  */
 
-#include "classifier_mac.h"
+#include "classifier.h"
 #include "forwarder.h"
 #include "shared/secondary/return_codes.h"
 #include "shared/secondary/json_helper.h"
-- 
2.17.1

